lib/compliance.js

```diff
@@ -59,16 +59,17 @@ function matchOfac (deviceId, customer) {
     })
 }
 
-function validateOfac (deviceId, customer) {
-  if (customer.sanctionsOverride === 'blocked') return false
-  if (customer.sanctionsOverride === 'verified') return true
+function validateOfac (deviceId, config, customer) {
+  if (!config.sanctionsVerificationActive) return Promise.resolve(true)
```

Comment:
Actually we need to return a Promise here otherwise at line 73 there won't be a thenable but a boolean

Reply:
👍